### PR TITLE
Turn off dropdown on mouse enter - it is very bad UX

### PR DESCRIPTION
**Pull request by jacmoe**
Bootstrap has expand on hover turned off by default.
And there is little reason to do this by javascript since it is fairly easy to do by css (display: block on hover).

It is quite annoying to have the navbar menu fly out when you accidentally touch it with your mouse pointer in my opinion.

### CODE REVIEW

File: less/roxie/js/custom.js [hunk 1]
```diff
@@ -43,15 +43,15 @@ $(window).scroll(function(){
...
+$(document).ready(function() {
 	//NAVBAR
-	$('.navbar-main-menu > li.dropdown').mouseenter(function() {
-		$(this).addClass('open');
-	});
-	
-	$('.navbar-main-menu > li.dropdown').mouseleave(function() {
-		$(this).removeClass('open');
-	});
+	//$('.navbar-main-menu > li.dropdown').mouseenter(function() {
```
Comment (samdark) [inline less/roxie/js/custom.js]: Don't comment. Remove.

Comment (jacmoe) [inline less/roxie/js/custom.js]: In that case, I will remove everything that is currently commented out because I agree.

Comment (samdark) [inline less/roxie/js/custom.js]: Yes. Do it.

**Comment by jacmoe** [comment]
Removed commented out code. :)

**Comment by samdark** [comment]
Thanks!

**Pull request merged**